[2022-09-05 03:08:00,199] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-09-02T00:00:00+00:00 [queued]>
[2022-09-05 03:08:00,332] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-09-02T00:00:00+00:00 [queued]>
[2022-09-05 03:08:00,332] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:08:00,333] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:08:00,333] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:08:00,405] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-09-02T00:00:00+00:00
[2022-09-05 03:08:00,410] {standard_task_runner.py:52} INFO - Started process 497 to run task
[2022-09-05 03:08:00,463] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-09-02T00:00:00+00:00', '--job-id', '106', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp5hd73ulx', '--error-file', '/tmp/tmp2naa3vzx']
[2022-09-05 03:08:00,490] {standard_task_runner.py:77} INFO - Job 106: Subtask second_task
[2022-09-05 03:08:00,712] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-09-02T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:08:01,028] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T00:00:00+00:00
[2022-09-05 03:08:01,031] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:08:01,037] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:08:01,085] {bash.py:169} INFO - Output:
[2022-09-05 03:08:01,100] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:08:01,101] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:08:01,262] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220902T000000, start_date=20220905T030800, end_date=20220905T030801
[2022-09-05 03:08:01,562] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:08:01,624] {local_task_job.py:146} INFO - Task exited with return code 0
